src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      class="form"
+      data-qa="big"
+    >
+      <label class="search_label search_label_big">
+        <input
+          type="text"
+          class="search search_input--big"
```

Comment:
So it will be better if you add classes similar to label tag. 
Update names at this tag and next one as well + css
```suggestion
          class="search_input search_input--big"
```